manifests/kustomize/third-party/grafana/grafana-deployment.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
```

Comment:
Can you add a README in grafana dir about where we can get these manifests and what customizations we've made to it?

Reply:
I've planned to add a readme for both prometheus and grafana. such as configure data source as prometheus in grafana. But on the other hand, guides like that are also available in numerous online tutorials that help people setup prometheus+grafana.